### PR TITLE
ibm-object-storage-plugin - update chart version and form view support

### PR DESCRIPTION
**Pull request by Bhagyashreek8**

### CODE REVIEW

**Comment by Bhagyashreek8** [comment]
@pedjak @morstad I have a raised a PR from my fork and the `Initial PR scan` seems to be failing. Earlier when I raised the PR, I did not hit with this issue. 
This seems to be an issue with github actions -https://github.community/t/github-actions-are-severely-limited-on-prs/18179#M9249

Could you please suggest on how to go about this ?

**Comment by Bhagyashreek8** [comment]
Tested and verified my changes on OCP 4.6 cluster.

File: stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml [hunk 1]
```diff
@@ -1,7 +1,6 @@
-{{- include "ibm-object-storage-plugin.licenseValidate" .  | required "License must be accepted by setting license key to true" }}
-
 ---
 # ServiceAccount for ibmcloud-object-storage-driver
+{{- include "ibm-object-storage-plugin.licenseValidate" .  | required "License must be accepted by setting license key to true" }}
```
Comment (pedjak) [inline stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml:3]: can you please revert this change, i.e. keep the line where it was - there is a known issue with GH page rendering if we move it after `---` line.

Comment (Bhagyashreek8) [reply stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml:3]: @pedjak I do hit with the below error during chart install, if I move the line above `---`. Due to which I had to move it after `---`
```
Error: YAML parse error on ibm-object-storage-plugin/templates/flex-driver-sa.yaml: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal bool into Go value of type releaseutil.SimpleHead

Comment (pedjak) [reply stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml:3]: hm, we needed to for example here: https://github.com/redhat-developer/redhat-helm-charts/commit/2281025e8e4b8df93f0446fd14104c0350234f0d to introduce this week, otherwise GH pages are not regenerated...

**Comment by pedjak** [comment]
can you please squash the commits?

File: stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml (hunk 1 above)
Comment (Bhagyashreek8) [reply stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml]: @pedjak I have reverted the above change and squashed the commits. Still, the PR scan seems to fail.

Comment (pedjak) [reply stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml]: there are still 2 commits in this PR, can you squash them into single one?

Comment (Bhagyashreek8) [reply stable/ibm-object-storage-plugin/templates/flex-driver-sa.yaml]: Done.

**Comment by Bhagyashreek8** [comment]
@pedjak I have squashed the commits into single commit. Could you let me know if any more review comments / changes required for this PR to be merged..
